include/openssl/objects.h

```diff
@@ -176,6 +176,12 @@ int OBJ_find_sigid_by_algs(int *psignid, int dig_nid, int pkey_nid);
 int OBJ_add_sigid(int signid, int dig_id, int pkey_id);
 void OBJ_sigid_free(void);
 
+/* This is to correct for the auditIdentity OID having been assigned the wrong
+name. */
+#define SN_ac_auditIdentity             "ac-auditIdentity"
+#define LN_ac_auditIdentity             LN_ac_auditEntity
+#define NID_ac_auditIdentity            NID_ac_auditEntity
+#define OBJ_ac_auditIdentity            OBJ_ac_auditEntity
```

Comment:
This avoids any API break however it also makes the `SN_ac_auditIdentity` useless - it will not be found in the object database. Not sure, if it is a problem or not.

Reply:
Yeah, I knew that would be the case. I thought I'd just add it for thoroughness. I am not sure how to change just the short name for `ac-auditEntity` without introducing a breaking change. Does the `Cname` macro in `objects.txt` do that?

Reply:
Unfortunately the Cname doesn't seem to be doing anything useful. @levitte might have some ideas perhaps how to do this?

Reply:
Well, we haven't heard back from @levitte . Should we just remove the short name line above? At least the original OID entry has a correct long name now, which is what shows up in the CLI outputs.